pkg/scorecard_result.go

```diff
@@ -116,7 +116,7 @@ func FormatResults(
 		// TODO: support config files and update checker.MaxResultScore.
 		err = results.AsSARIF(opts.ShowDetails, log.ParseLevel(opts.LogLevel), os.Stdout, doc, policy, opts)
 	case options.FormatJSON:
-		err = results.AsJSON2(opts.ShowDetails, log.ParseLevel(opts.LogLevel), doc, os.Stdout)
+		err = results.AsJSON3(opts, log.ParseLevel(opts.LogLevel), doc, os.Stdout)
```

Comment:
I'd be hesitant with this change. I thought this was enabling the `JSONScorecardResultV3` for the `--format json` users. 

This is misleading in my opinion.